packages/o365/data_stream/audit/elasticsearch/ingest_pipeline/default.yml

```
@@ -326,7 +326,7 @@ processors:
       if: ctx.event?.code == "AzureActiveDirectoryStsLogon"
   - append: 
       field: event.type
-      value: "authentication_{{event.outcome}}"
+      value: access
       if: 'ctx.event?.code == "AzureActiveDirectoryStsLogon" && ctx.event?.outcome != null && ctx.event?.outcome != "unknown"'
```

```suggestion
      if: 'ctx.event?.code == "AzureActiveDirectoryStsLogon"
```

I guess the other two conditions aren't required now ?

---

Eeep. Not quite right.